docs/en_us/internal/testing.rst

```diff
@@ -610,6 +616,24 @@ During acceptance test execution, Django log files are written to
 
 **Note**: The acceptance tests can *not* currently run in parallel.
 
+
+Testing internationalization with dummy translations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Any text you add to the platform should be internationalized. To generate
+translations for your new strings, run the following:
+
+::
+
+    paver i18n_dummy
+
+This will generate dummy translations for the individual language files.
+You can preview the dummy languages on your local machine and also in
+your sandbox, if and when you create one.
+
+Do not commit the ``.po``, ``.mo``, translation-related ``.js`` files!
```

Comment:
Does this sentence refer to the files that contain the dummy translations for testing? If yes, it would be good to make that connection clear.